src/safe_apps/tests/test_signals.py

```diff
@@ -410,6 +410,9 @@ def test_on_tag_delete_with_safe_app(self) -> None:
             "utf-8"
         )
 
+    # Otherwise fails when testing with all suites
```

Comment:
I've left a TODO to investigate this issue (already commented out in line 525, where I added a TODO as well).